WaysofWork.md

```diff
@@ -65,6 +68,8 @@ Stories should not be created without consensus, if you DO NEED to create one to
 
 It is the responsibility of the Accountable team or person, with help from the RPM where necessary, to create and complete a story.
 
+Active stories are expected to be updated at least once a month via a comment on their progress
```

Comment:
I suggest we encourage this in order to build the monthly report while not asking for too many updates in between (any changes in issues mentioned should be already visible).
It also means that RPM+tech leads have a task of going through stories.
Asking for contributions directly on a monthly report could mean stories are not properly updated.

Reply:
Outdated, I have modified to reflect the RPM will update based on weekly (the weekly description needs updating in turn)